python/pyspark/sql/dataframe.py

```diff
@@ -520,11 +520,14 @@ def collect(self):
 
     @ignore_unicode_prefix
     @since(2.0)
-    def toLocalIterator(self):
+    def toLocalIterator(self, prefetchPartitions=False):
```

Comment:
looks like you miss to pass prefetchPartitions to self._jdf.toPythonIterator() below?

Reply:
Good catch :)